OutOfSchool/OutOfSchool.WebApi/Services/IWorkshopService.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using OutOfSchool.WebApi.Models;
+
+namespace OutOfSchool.WebApi.Services
+{
+    /// <summary>
+    /// Interface of WorkshopService.
```

Comment:
this summary should be more informative
user of this interface should be able to get the purpose of it from the description